src/pages/UserTable/ChangeUserRoleForm.tsx

```diff
@@ -88,15 +88,20 @@ const ChangeUserRoleForm = ({ record, setShowModal, onChange, user }: Props) =>
             },
           ]}
         >
+
       <Select onChange={handleChange}>
         <Option value={Roles.Supporter} disabled={handleDisabled()}>Прихильник</Option>
         <Option value={Roles.PlastMember} disabled={handleDisabled()}>
           Дійсний член організації
         </Option>
         <Option value={Roles.FormerPlastMember} disabled={handleDisabled()}>Колишній член Пласту</Option>
-        <Option value={Roles.RegisteredUser} disabled={disabled}>Зареєстрований користувач</Option>
-        
+        { user.userRoles != Roles.FormerPlastMember ? (
```

Comment:
Try this:
`user.userRoles == Roles.FormerPlastMember ? then you template`

Should work the same way

Reply:
fixed!